doc/source/whatsnew/v2.2.0.rst

```diff
@@ -14,10 +14,27 @@ including other versions of pandas.
 Enhancements
 ~~~~~~~~~~~~
 
-.. _whatsnew_220.enhancements.enhancement1:
+.. _whatsnew_220.enhancements.calamine:
 
-enhancement1
-^^^^^^^^^^^^
+Calamine engine for :func:`read_excel`
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The ``calamine`` engine was added to :func:`read_excel`.
+It's used 'python-calamine', that binging for Rust library `calamine <https://crates.io/crates/calamine>`__.
```

Comment:
I think the grammar is off here. Something like:

`` It uses `python-calamine`, which provides Python bindings for the Rust library... ``